website/docs/index.html.markdown

```diff
@@ -139,6 +139,19 @@ variables in order to keep credential information out of the configuration.
   that Terraform can be tricked into writing secrets to a server controlled
   by an intruder. May be set via the `VAULT_SKIP_VERIFY` environment variable.
 
+* `create_intermediate_child_token` - (Optional) Set this to `false` to disable
```

Comment:
For extra clarity it may be a good idea to update the `token` documentation which discusses "child" token creation.

Reply:
updated :+1: